adr-001-rand-patch.md

```
@@ -0,0 +1,79 @@
+```toml
+# Pinned due to RUSTSEC-2026-0097: rand 0.9.2 is unsound via proptest
+# This only affects dev/test dependencies (property-based tests), not production.
+[patch.crates-io]
+rand = "0.9.4"
+```
+
+This patch ensures that whenever Cargo resolves `rand` for any dependency (including transitive), it substitutes `rand 0.9.4` regardless of what version `proptest` requests.
+
+## Consequences
+
+### Positive
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)